.github/workflows/npm_release.yml

```diff
@@ -0,0 +1,47 @@
+name: Do release
```

Comment:
can we please discuss about a github action release feature in another pull request and remove it for now?